src/hangar_sim/launch/sim/localization_launch.py

```
@@ -26,47 +26,42 @@
     bringup_dir = get_package_share_directory("nav2_bringup")
 
     namespace = LaunchConfiguration("namespace")
     map_yaml_file = LaunchConfiguration("map")
     use_sim_time = LaunchConfiguration("use_sim_time")
     autostart = LaunchConfiguration("autostart")
     params_file = LaunchConfiguration("params_file")
-    use_composition = LaunchConfiguration("use_composition")
     container_name = LaunchConfiguration("container_name")
     container_name_full = (namespace, "/", container_name)
-    use_respawn = LaunchConfiguration("use_respawn")
     log_level = LaunchConfiguration("log_level")
```

Flake8 (7.3.0)

[error] local variable 'log_level' is assigned to but never used

(F841)